### DISCUSSION TITLE
Wrong link-thumbnail markup/format in article body?

### DISCUSSION THREAD

**Discussion by turbodude** [martinrotter/rssguard#1084] [Q&A]
The feed that I'm subscribed to has this markup:

```
<a href="full-image.jpg" target="_blank" rel="noopener">
	<img src="thumbnail.jpg" decoding="async" />
</a>
```
When the link/image in RSSGuard gets clicked the thumbnail is opened in the browser instead of the full image. 

Not sure if this is a bug or something tweakable in the settings, but I couldn't figure out how to change that.

Version: 4.5.0 (built on Windows/AMD64)
Revision: 7175578b6-nowebengine
Build date: 24/08/2023 11:57
Qt: 6.5.2 (compiled against 6.5.2)

**Comment by martinrotter** [comment]
Provide link to feed which has this behavior pls.

**Reply by turbodude** [reply]
`https://pornrips.to/category/720p/feed/`

QuiteRSS had no issues with it by the way
